config.template.yml

```diff
@@ -419,40 +418,40 @@ session:
 
     ## The Redis TLS configuration. If defined will require a TLS connection to the Redis instance(s).
```

Comment:
If you check this section in your editor, the indent of server_name, skip_verify, and minimum_version was broken. I'd suggest copying it from master.

Sometimes editors will want to auto correct the indent when you paste and you can undo that with CTRL-Z